### PR TITLE
Update QUEST.tsv [Final Pass 2/2]

### PR DESCRIPTION
**Pull request by ghost**
Final pass of QUEST.tsv. Hope this wraps it all up! :D

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -2118,7 +2118,7 @@ QUEST_20150714_002121	The pot of silence is at Memum Corridor… {nl}I'm so… t
 QUEST_20150714_002122	$Where to start…
 QUEST_20150714_002123	$I was a disciple of Agailla Flurry when I was alive. {nl}I didn't have any magical abilities so I couldn't use magic.
 QUEST_20150714_002124	$That's why I survived Helgasercle's wrath. {nl}He must've thought he didn't have to kill me since I wasn't able to use magic.
-QUEST_20150714_002125	$I am confident I am better when it comes to theories than other disciples.. {nl}That's why I studied magic. And in my studies, I learned of the magic stone of interaction.
+QUEST_20150714_002125	{memo X}$I am confident I am better when it comes to theories than other disciples… {nl}That's why I studied magic. And in my studies, I learned of the magic stone of interaction.
```
Comment (Soukyuu) [inline QUEST.tsv]: Are you sure you should be adding memo X here? I'd rather leave it to imc.

File: QUEST.tsv [hunk 2]
```diff
@@ -2249,31 +2249,31 @@ QUEST_20150717_002252	What do you think? They make a pretty good friend, right?
 QUEST_20150717_002253	Item Merchant 
 QUEST_20150717_002254	I'm sorry I can't come forward myself. {nl}But the demons will sense it when I use the spell.
 QUEST_20150717_002255	I will fix the twisted spell of the tower.{nl} I will have to use the spell at first and monsters will rush in. Please protect me.
-QUEST_20150717_002256	I hope Gabija is safe… Let's move fast.
+QUEST_20150717_002256	$I hope Gabija's safe… Let's move fast.
```
Comment (Sunaries) [inline QUEST.tsv]: In my opinion the previous sentence is better.

Comment (Soukyuu) [inline QUEST.tsv]: I tend to agree. Contractions might make sentences more compact, but we shouldn't make things too colloquial.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**